### PR TITLE
Give importance to rails command [ci skip]

### PR DESCRIPTION
**Pull request by mechanicles**
Give importance to rails command in the comment section.

### CODE REVIEW

File: railties/lib/rails/generators/rails/app/templates/app/assets/javascripts/cable.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Action Cable provides the framework to deal with WebSockets in Rails.
-// You can generate new channels where WebSocket features live using the rails generate channel command.
+// You can generate new channels where WebSocket features live using the 'rails generate channel' command.
```
Comment (maclover7) [inline railties/lib/rails/generators/rails/app/templates/app/assets/javascripts/cable.js]: Can you change the `'`s to backticks ?

Comment (guilleiguaran) [inline railties/lib/rails/generators/rails/app/templates/app/assets/javascripts/cable.js]: 👍

**Comment by mechanicles** [comment]
@maclover7 Updated it and also added the same change in another similar file too.

File: railties/lib/rails/generators/rails/app/templates/app/assets/javascripts/cable.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Action Cable provides the framework to deal with WebSockets in Rails.
-// You can generate new channels where WebSocket features live using the rails generate channel command.
+// You can generate new channels where WebSocket features live using the `rails generate channel` command.
```
Comment (vipulnsward) [inline railties/lib/rails/generators/rails/app/templates/app/assets/javascripts/cable.js]: Changes to template comments should not be ci-skipped. Can you remove that from commit?

**Pull request merged**